lib/rules/forbid-prop-types.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
   meta: {
     docs: {
       description: 'Forbid certain propTypes',
-      category: 'Best Practices',
+      category: 'suggestion',
```

Comment:
i prefer eslint's previous mechanism for categorizing rules, to be honest. is there any benefit from using this style?

Reply:
Do you mean categorizing rules as problems, best practices, and stylistic issues? In that case I agree with you.

Reply:
Yes. I'm saying that i think eslint's old categories were superior, and it's better to stick with those :-)

Reply:
Updated.